packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumb.ts

```diff
@@ -14,21 +15,34 @@ import type { BreadcrumbProps, BreadcrumbState } from './Breadcrumb.types';
 export const useBreadcrumb_unstable = (props: BreadcrumbProps, ref: React.Ref<HTMLElement>): BreadcrumbState => {
   const {
     appearance = 'transparent',
+    disableFocus,
     dividerType = 'chevron',
     iconPosition = 'before',
     size = 'medium',
+    list,
     ...rest
   } = props;
+
+  const focusAttributes = !disableFocus
```

Comment:
React Hooks can't be called conditionally, so incase the `disableFocus` prop changes, React will break. Try to store the values of the `useArrowNavigationGroup`. :)

Reply:
fixed